borrowing/views.py

```diff
@@ -19,10 +20,12 @@ class BorrowingViewSet(
     permission_classes = (IsAuthenticated,)
 
     def get_queryset(self):
-        if self.request.user.is_staff:
-            return self.queryset
+        queryset = self.queryset.select_related("book")
 
-        return Borrowing.objects.filter(user=self.request.user)
+        if self.action == "list" and not self.request.user.is_staff:
+            return Borrowing.objects.filter(user=self.request.user)
```

Comment:
Better to write `queryset = Borrowing.objects.filter...`, bc you don't use select_related in this way